GUIDE.md

```diff
@@ -2,109 +2,86 @@
 
-	$ fingerprint scan /bin
-	fingerprint scan /bin | head -n 100
-	C /bin
-		fingerprint.version 2.1.3
-		options.checksums MD5, SHA2.256
-		options.extended false
-		summary.time.start 2016-07-01 15:16:51 +1200
-	D 
-	F [
-		file.size 22464
-		key.MD5 576c4bb1e8cd269d3bdaf2b6dcd62cda
-		key.SHA2.256 8cd75d8ddbeee27938c5fb9dee380bbc40159be687ba18d99d5b10ef3b16a7cb
-	F bash
-		file.size 628496
-		key.MD5 5d7583d80e5314ac844eedc6d68c6cd7
-		key.SHA2.256 fe162e301556121782e1e5334a023e94f742a3a66434812620ae41a5da5f3360
-	F cat
-		file.size 23520
-		key.MD5 3fb0e3ca64776d182c422400a09673c3
-		key.SHA2.256 2b4e43b4fe74e662a8063b61ab85c65d3a5d6f431166c38abb1f9c2df06ac69c
-	F chmod
-		file.size 33904
-		key.MD5 ecb64579c6dd0ebee31bf8e4d4cdcc6e
-		key.SHA2.256 139432e4cfdd8761ab8ddd36f0121f8d34ef4b05e61d63d45e8703ef53a2b447
-	F cp
-		file.size 28832
-		key.MD5 a8ebcee2d17317beee2136ec59bfba4d
-		key.SHA2.256 c126b5445251cb5be295d6a9e238bb71df9e8450dee1e84c1ee46760052985ba
-	F csh
-		file.size 378624
-		key.MD5 0d3ea03a76f98011f35812a912068515
-		key.SHA2.256 c20dac3882d08111cafcc42b4e689b8d963650244f8b8e9e13ade7cd9300d538
-	F date
-		file.size 28368
-		key.MD5 52c4e2ef8d1e621901abaa75d875d743
-		key.SHA2.256 6a6cade84d420801e974aa493b127a1af1bb9482e9e9fc8a563fb626c1f5d3c7
-	F dd
-		file.size 31856
-		key.MD5 88cb8607654c414248b629441f74e865
-		key.SHA2.256 dc53fcea0dda988dc63225b06c7c5b3690aa3926947b8d3ee15362c008c9f5c4
-	F df
-		file.size 27440
-		key.MD5 f0857735cd9e35b183204528f2be2114
-		key.SHA2.256 f71696bc4933373a1b64f5510a98909752256f1c57122999cf5784c30f2f8216
-	-- snip --
-	F zsh
-		file.size 573600
-		key.MD5 52d52aa8579eaef58addc0757f9c97d9
-		key.SHA2.256 72508fe3cef985ca91b766727c4d9a6374f216a8b69a13ce41677f2e0400eacd
-	S 37 files processed.
-		summary.directories 1
-		summary.excluded 0
-		summary.files 37
-		summary.size 4983584
-		summary.time.end 2016-07-01 15:17:26 +1200
+``` text
+$ fingerprint scan /tmp/test
+C /tmp/test
+	fingerprint.version 2.1.5
+	options.checksums SHA2.256
+	options.extended false
+	summary.time.start 2016-12-09 16:42:41 -0800
+D
+F file-1.txt
+	file.size 0
+	key.SHA2.256 e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+F file-2.txt
+	file.size 0
+	key.SHA2.256 e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+F file-3.sh
+	file.size 0
+	key.SHA2.256 e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+S 3 files processed.
+	summary.directories 1
+	summary.excluded 0
+	summary.files 3
+	summary.size 0
+	summary.time.end 2016-12-09 16:42:41 -0800
+```
 
 ## Verifing Fingerprints
 
 Files can be checked against a given fingerprint. In the case of `verify`, only changes and deletions will be reported. Additions are not reported.
 
 To generate a fingerprint for a given path:
 
-	fingerprint scan /bin > bin.fingerprint
+	fingerprint scan /tmp/test > /tmp/test.fingerprint
```

Comment:
To eliminate the need for the 'snip' where you truncated the fingerprint output, and to show examples that show the new SHA256 default, and not MD5.